bin/welcome.js

```diff
@@ -0,0 +1,10 @@
+#! /usr/env/bin node
+
+const chalk = require( 'chalk' );
+
+console.log( chalk.cyan( '             _                           ' ) );
+console.log( chalk.cyan( '    ___ __ _| |_   _ _ __  ___  ___      ' ) );
+console.log( chalk.cyan( '   / __/ _\` | | | | | \'_ \\/ __|/ _ \\ ' ) );
+console.log( chalk.cyan( '  | (_| (_| | | |_| | |_) \\__ \\ (_) |  ' ) );
+console.log( chalk.cyan( '   \\___\\__,_|_|\\__, | .__/|___/\\___/ ' ) );
+console.log( chalk.cyan( '               |___/|_|                \n' ) );
```

Comment:
@dmsnell This is what I see in my machine (Windows 10, Chrome):
![calypso-art](https://user-images.githubusercontent.com/1715800/26905121-5619df8a-4bdd-11e7-949a-d26d3f54ca6b.png)
The ASCII art should be ASCII, specially if we are pushing for compatibility with lesser shells like Windows' `cmd.exe`. In other words: if *I* can't have emojis in my terminal, neither will any of you! 😂